services/spar/src/Spar/API/Util.hs

```diff
@@ -6,6 +6,7 @@
 {-# LANGUAGE ScopedTypeVariables   #-}
 {-# LANGUAGE PolyKinds             #-}
 
+-- | Swagger utilities.
```

Comment:
I guess that means a better place for this module would be `Spar.API.Swagger.Util`.  Oh well.